### DISCUSSION TITLE
create_model doesn't show scores for test data

### DISCUSSION THREAD

**Discussion by shreeprasadbhat** [pycaret/pycaret#3174] [General]
pycaret create_model function doesn't show scores for test data, when cross_validation is enabled.

This is my setup with train_size=0.8, pycaret will split df_train into 80% train and 20% test.
```clf = setup(data=df_train,target='label',train_size=0.8)```

I create a xgboost model with cross_validation=True.
```xgboost = create_model('xgboost', cross_validation=True, fold=10, return_train_score=True)```

Below is the output of create_model function. 

<img width="328" alt="image" src="https://user-images.githubusercontent.com/11194928/207262330-c9347b76-faec-4c7e-b22a-500e7682b85e.png">

I am assuming that 10-fold CV is done on 80% of train. Then where is the scores for 20% test data?

**Comment by turkalpmd** [comment]
From my understanding, creating a model only uses the training data. However, you may be able to use the predict() module to predict the other 20% of your data. It's worth noting that the train_size argument of the setup module is used for validation of your data, and the test_data argument is set to None by default.
